test/library/packages/ProtobufProtocolSupport/endToEndRunner.skipif

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env python
+
+from distutils.spawn import find_executable
+
+if find_executable("protoc") is not None and find_executable("protoc-gen-chpl") is not None:
```

Comment:
Note to mentors: need to figure out a way to get this in our nightly testing set up - we'll need to install protoc somewhere, etc.